### PR TITLE
zh-CN: create Glossary/Page_prediction

### PR DESCRIPTION
**Pull request by PassionPenguin**

### CODE REVIEW

File: files/zh-cn/glossary/page_prediction/index.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
+---
+title: Page prediction
```
Comment (yin1999) [inline files/zh-cn/glossary/page_prediction/index.md:2]:
```suggestion
title: 页面预测
```

File: files/zh-cn/glossary/page_prediction/index.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+slug: Glossary/Page_prediction
+l10n:
+  sourceCommit: ada5fa5ef15eadd44b549ecf906423b4a2092f34
+---
+
+{{GlossarySidebar}}
+
+**页面预测**是一种浏览器功能或是一段脚本，在启用时会告诉浏览器在用户请求内容之前下载用户可能访问的资源。页面预测通过使预测内容几乎瞬间加载来提高性能。然而，页面预测也可能会下载用户并不需要的内容。
```
Comment (yin1999) [inline files/zh-cn/glossary/page_prediction/index.md:10]: 第二句话的翻译读着怪怪的

File: files/zh-cn/glossary/page_prediction/index.md [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+
+一些网页应用程序就包括了预测功能，根据浏览历史和相关搜索自动完成搜索文本和地址栏 URL。例如，当用户在地址栏输入内容时，浏览器可能会在用户提交请求之前将地址栏中的当前文本发送给搜索引擎。
```
Comment (yin1999) [inline files/zh-cn/glossary/page_prediction/index.md:12]:
```suggestion
一些 Web 应用程序就包括了预测功能，根据浏览历史和相关搜索自动完成搜索文本和地址栏 URL。例如，当用户在地址栏输入内容时，浏览器可能会在用户提交请求之前将地址栏中的当前文本发送给搜索引擎。
```

File: files/zh-cn/glossary/page_prediction/index.md [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+
+尽管浏览器页面预测和预测服务可以加快页面加载速度，但它们会消耗额外的带宽。此外，预加载的网站和嵌入的内容即使没有被访问也可以设置和读取它们的 cookie。
+
+## 参见
+
+- [预渲染](/zh-CN/docs/Glossary/Prerender)
+- [预获取](/zh-CN/docs/Glossary/Prefetch)
```
Comment (yin1999) [inline files/zh-cn/glossary/page_prediction/index.md:19]:
```suggestion
- [预取](/zh-CN/docs/Glossary/Prefetch)
```

File: files/zh-cn/glossary/page_prediction/index.md (hunk 2 above)
Comment (PassionPenguin) [reply files/zh-cn/glossary/page_prediction/index.md:10]: Page prediction improves performance by enabling almost instant loading of predicted content.

（原句太太太短了，有点不知其所以然，要好好思考一下才能懂……翻译出来的结果自然也不好）

Comment (PassionPenguin) [reply files/zh-cn/glossary/page_prediction/index.md]: 我的想法是修改原文➕前提补充

Page Prediction is a browser feature or script which, when enabled, tells the browser to download resources the user is likely to visit before the user requests the content. *As the content is pre-downloaded, page prediction allows almost instant loading of predicted content, thus improving performance.*

**Review by yin1999 — APPROVED** [review]

**Pull request merged**